src/driver/index.js

```diff
@@ -1,9 +1,11 @@
-const { events, chance } = require('../utils');
-const { io } = require('socket.io-client');
+const { EVENT_NAMES, chance } = require('../utils');
+
+const { io } = require("socket.io-client");
+const events = io("ws://localhost:3333"); 
```

Comment:
Make sure your port matches the others(hub and vendors)

Reply:
Fixed-- thank you!